### DISCUSSION TITLE
Codespaces not working and open

### DISCUSSION THREAD

**Discussion by iasdomingos03out** [community/community#123312] [Codespaces]
### Select Topic Area

Bug

### Body

Hello. Today I tried open my codespace and received this message. 
![image](https://github.com/community/community/assets/166149821/f30c230b-512b-43b0-b868-c8369f0df541)
I can't save, enter, or perform anything in any of the codespaces.
Days ago I had other problems and not can access the codespace.

Can help me please?

**Comment by rongxin-liu** [comment]
If you visit https://github.com/codespaces, stop your current codespace, then start it again. Does it fix the issue?

**Reply by iasdomingos03out** [reply]
Hi.It's me again. In this day, I tried start it again, change the network and browser and unfortunately not work.

**Reply by iasdomingos03out** [reply]
In some days I don't have problems.So I don't know what it could be. But I thank for your attention.
